site/docs/concepts/connectors.md

```diff
@@ -286,7 +288,7 @@ In some cases, your source or destination endpoint may be within a secure networ
 to allow direct access to its port due to your organization's security policy.
 
 :::tip
-If permitted by your organization, a quicker solution is to whitelist the Estuary IP address, `34.121.207.128`.
+If permitted by your organization, a quicker solution is to whitelist the Estuary IP addresses, `34.121.207.128, 35.226.75.135, 34.68.62.148`.
```

Comment:
I think from a UI/UX aspect breaking these up into a small list that has one IP per line could make sense. However, that would make them a lot harder to copy/paste the list.

From an ops perspective would keeping these as a comma separated list make it easier to configure on the client's system?

![image](https://github.com/user-attachments/assets/5a4b0a21-a92a-484f-9f0d-7199bb8e0ca5)

Reply:
![image](https://github.com/user-attachments/assets/3da7d162-7598-4911-ac68-309008760570)

This is actually kind of tricky. I moved these into a code block for where it's in a tool tip and it looks like the attached. 

In other cases, I just added a `:` so it was clear a list was coming. Trying to make a UL in markdown for other cases was very visually jarring. The other option was to put something like 


`34.121.207.128, 35.226.75.135` and `34.68.62.148` 

for proper English, but the flow on that is gross too.

Reply:
Looks good to me and follows the pattern up above... though that is kinda different given you _really_ should copy the value.

![image](https://github.com/user-attachments/assets/8dc3ebac-6a50-441c-bfe4-59078bb173bf)